pep-0484.txt

```diff
@@ -610,6 +611,21 @@ the runtime class of the objects created by instantiating them doesn't
 record the distinction. This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
 
+Class attribute of a generic class can be looked up only through the instance
+of the class; any attempt to use generic classes (parametrized or not) to
+access attributes will result in both type check failure and runtime error::
+
+  # (continued from previous example)
+  Node[int].x = 1  # Error
+  Node[int].x      # Error
+  Node.x = 1       # Error
+  Node.x           # Error
+  type(p).x        # Error
+  p.x              # Ok
+  Node[int]().x    # Ok
+
+In addition, ``ClassVar`` cannot be parametrized.
```

Comment:
Oh I thought so too, but I searched the text and found it uses `parametrized`. Now I searched again, and I found it uses both, 2 times each (not counting my own usage) 😆 I guess I should have checked dictionary instead ...